src/pudl/etl/epacems_assets.py

```diff
@@ -115,36 +115,32 @@ def consolidate_partitions(context, partitions: list[YearPartitions]) -> None:
         context: dagster keyword that provides access to resources and config.
         partitions: Year and state combinations in the output database.
     """
-    partitioned_path = _partitioned_path()
-    # Create monolithic output in base output directory and parquet directory
-    monolithic_paths = [PudlPaths().output_dir, PudlPaths().output_dir / "parquet"]
+    partitioned_path = PudlPaths().output_dir / "core_epacems__hourly_emissions"
```

Comment:
Oh I thought having this under the `parquet/` directory made good organizational sense.  Or even just writing all the quarterly files directly out to the `parquet/` directory without the subdirectory would be fine, since we iterate over them by filename, rather than by walking the directory and there's "only" ~100 of them.  That way any parquet file we generate can be found in the same way without needing to think about whether it's in the special partitioned category.

Reply:
Oh yeah it was still writing the partitioned outputs to the `parquet/` directory, but was trying to read from the wrong place. Fixed now